src/App.js

```diff
@@ -26,20 +27,21 @@ const alex = {
 const App = () => (
   <div className="App">
     <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">My wife&apos;s name is Natasha</p>
+     <Person 
+       {...misha}
+     />
```

Comment:
```suggestion
     <Person {...misha} />
```
Коли пропсів менше трьох, можна писати в один рядок